src/reducers/__tests__/ui.test.js

```diff
@@ -423,6 +411,34 @@ test('finish dragging column divider', () => {
   });
 });
 
+test('openProjectPickerModal opens project picker modal', () => {
+  expect(applyActions(openProjectPickerModal())).toMatchObject({
+    isProjectPickerModalOpen: true,
+  });
+});
+
+test('closeProjectPickerModal closes project picker modal', () => {
+  applyActions(openProjectPickerModal());
+  expect(applyActions(closeProjectPickerModal())).toMatchObject({
+    isProjectPickerModalOpen: false,
+  });
+});
+
+describe('filterProjects filters', () => {
+  it('active projects', () => {
```

Comment:
We typically use `test` instead of `it` (especially if the test description isn’t forming a sentence after the “it”)